src/ui/layout/tutorial.md

```diff
@@ -1,237 +1,386 @@
+                    color: Colors.grey[500],
+                  ),
+                ),
+              ],
             ),
-          ],
-        ),
-      ),
-      /*3*/
-      Icon(
-        Icons.star,
-        color: Colors.red[500],
+          ),
+          /*3*/
+          Icon(
+            Icons.star,
+            color: Colors.red[500],
+          ),
+          const Text('41'),
+        ],
       ),
-      const Text('41'),
-    ],
-  ),
-);
+    );
+  }
+}
 ```
 
 {:.numbered-code-notes}
- 1. Putting a `Column` inside an `Expanded` widget stretches
-    the column to use all remaining free space in the row.
-    Setting the `crossAxisAlignment` property to
-    `CrossAxisAlignment.start` positions the column at
-    the start of the row.
- 2. Putting the first row of text inside a `Container`
-    enables you to add padding. The second child in the
-    `Column`, also text, displays as grey.
- 3. The last two items in the title row are a star icon,
-    painted red, and the text "41". The entire row is in
-    a `Container` and padded along each edge by 32 pixels.
-    Add the title section to the app body like this:
 
-<?code-excerpt path-base="layout/lakes"?>
-<?code-excerpt "{../base,step2}/lib/main.dart" from="return MaterialApp"?>
+1. To use all remaining free space in the row, use the `Expanded` widget to
+   stretch the `Column` widget.
+   To place the column at the start of the row,
+   set the `crossAxisAlignment` property to `CrossAxisAlignment.start`.
+2. To add space between the rows of text, put those rows in a `Padding` widget.
+3. The title row ends with a red star icon and the text `41`.
+    The entire row falls inside a `Padding` widget and pads each edge
+    by 32 pixels.
+
+### Change the app body to a scrolling view
+
+In the `body` property, replace the `Center` widget with a
+`SingleChildScrollView` widget.
+Within the `SingleChildScrollView` widget, replace the `Text` widget with a
+`Column` widget.
+
+<?code-excerpt "{../base,step2}/lib/main.dart" from="body:" to="children: ["?>
 ```diff
 --- ../base/lib/main.dart
 +++ step2/lib/main.dart
-@@ -14,11 +48,13 @@
-     return MaterialApp(
-       title: 'Flutter layout demo',
-       home: Scaffold(
-         appBar: AppBar(
-           title: const Text('Flutter layout demo'),
-         ),
+@@ -21,2 +17,3 @@
 -        body: const Center(
 -          child: Text('Hello World'),
-+        body: Column(
-+          children: [
-+            titleSection,
-+          ],
-         ),
-       ),
-     );
++        body: const SingleChildScrollView(
++          child: Column(
++            children: [
+```
+
+These code updates change the app in the following ways.
+
+* A `SingleChildScrollView` widget can scroll.
+  This allows elements that don't fit on the current screen to display.
+* A `Column` widget displays any elements within its `children` property
+  in a stack.
+  The first element listed in the `children` array displays at
```

Comment:
Here and other instances, do use "`children` list" instead of "array" since that's the Dart terminology and the actual type of `children`.

"array" is generally a more technical term as well.